Sources/Penny/Services/FaqsService/DefaultFaqsService.swift

```diff
@@ -10,7 +10,7 @@ import NIOHTTP1
 
 actor DefaultFaqsService: FaqsService {
 
-    let httpClient: HTTPClient = .shared
+    var httpClient: HTTPClient!
```

Comment:
IUO?

Reply:
There is a reason although i don't remember it off-hand.

Reply:
They are likely initialized first, then populated with the http client.